app/models/project.rb

```diff
@@ -22,6 +23,10 @@ class Project < ApplicationRecord
   scope :public_and_not_forked,
         -> { where(project_access_type: "Public", forked_project_id: nil) }
 
+  scope :public_access, -> { where(project_access_type: "Public") }
```

Comment:
```suggestion
  scope :public, -> { where(project_access_type: "Public") }
```
What about just public? It reads better for me